apps/metadata-editor-e2e/src/e2e/edit.cy.ts

```diff
@@ -1,12 +1,49 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+
 describe('editor form', () => {
-  beforeEach(() => {
+  let recordUuid
+  before(() => {
     cy.login('admin', 'admin', false)
 
-    // Alpine convention record
-    cy.visit('/edit/8698bf0b-fceb-4f0f-989b-111e7c4af0a4')
-
     cy.clearRecordDrafts()
+    // Duplicate a record to manipulate it at will
+    cy.visit('/catalog/search')
+    cy.get('.table-header-cell').eq(1).click()
+    cy.get('.table-header-cell').eq(1).click()
+    cy.get('[data-cy="table-row"]')
+      .first()
+      .find('[data-test="record-menu-button"]')
+      .click()
+    cy.get('[data-test="record-menu-duplicate-button"]').click()
+    cy.wait(1200)
```

Comment:
As duplication uses the draft system, can't we wait for the local storage to contain the wanted record?

Reply:
Well we would still have to wait for the draft saving to kick, right? And that would add many extra commands (which take time to execute) so I'm not sure it would be worth it...